src/test/java/bio/terra/service/dataset/DatasetDaoTest.java

```diff
@@ -56,7 +56,12 @@
 
     private UUID createDataset(DatasetRequestModel datasetRequest, String newName) {
         datasetRequest.name(newName).defaultProfileId(billingProfile.getId().toString());
-        return datasetDao.create(DatasetJsonConversion.datasetRequestToDataset(datasetRequest));
+        Dataset dataset = DatasetJsonConversion.datasetRequestToDataset(datasetRequest);
+        dataset.getTables().forEach(t -> {
+            t.rawTableName(t.getName() + "_raw");
+            t.softDeleteTableName(t.getName() + "_sd");
+        });
+        return datasetDao.create(dataset);
```

Comment:
I wonder if this and the earlier change in `CreateDatasetMetadataStep` could benefit from a `DatasetUtils` class which would have a few of these types of helper methods.

Reply:
That was one thought. The others I had were:
1. Make a `static` method on `Dataset` itself, so it's more obvious.
2. Update the existing definition of `DatasetJsonConversion.tableModelToTable` to generate & inject the names, so the returned Dataset is immediately reusable.

Point 1 doesn't seem to match the conventions in the codebase, at least as far as I can find. Point 2 is appealing because all of the existing calls "just work", but it's also gross because "converting" the same JSON payload twice won't give the same results.

Reply:
Updating from in person discussion: adding a new `DatasetUtils` class that can accomplish this would be useful. We can isolate this logic into one place and reference it from PDAO code + tests.